### PR TITLE
Changed include tag

### PR DESCRIPTION
**Pull request by Allexander203**

### CODE REVIEW

File: tcms/bugs/templates/bugs/search.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
                     </span>
                 </div>
             </div>
-            {% include "datetimepicker_script.html" with selector="#id_after" %}
+            {% include "include\datetimepicker_script.html" with selector="#id_after" %}
```
Comment (atodorov) [inline tcms/bugs/templates/bugs/search.html:26]: Use forward slash characters -> `/`

File: tcms/testcases/models.py [hunk 2]
```diff
@@ -262,14 +262,14 @@ class BugSystem(TCMSActionModel):
     """
 
     name = models.CharField(max_length=255, unique=True)
-    tracker_type = models.CharField(
+    tracker_type = models.CharField(  # pylint:disable=form-field-help-text-used
```
Comment (atodorov) [inline tcms/testcases/models.py:265]: Rebase to remove old commits

File: tcms/templates/datetimepicker_script.html [hunk 3]
```diff
@@ -1,13 +0,0 @@
-<script>
```
Comment (atodorov) [inline tcms/templates/datetimepicker_script.html:1]: File is deleted but the new file in `tcms/templates/include/datetimepicker_script.html` is not added to this pull request.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Review by atodorov — CHANGES_REQUESTED** [review]
Previously requested changes still not implemented.

**Review by atodorov — CHANGES_REQUESTED** [review]
Squash all commits together.

**Review by atodorov — DISMISSED** [review]
@asankov diff looks good to me. Feel free to merge when you are happy with how the commit log looks like.

**Review by asankov — APPROVED** [review]

**Pull request merged**